pkg/core/blockchain.go

```diff
@@ -504,6 +504,12 @@ func (bc *Blockchain) persist() error {
 			"blockHeight":     bHeight,
 			"took":            time.Since(start),
 		}).Info("blockchain persist completed")
+
+		// update monitoring metrics
+		blockHeight.Set(float64(bHeight))
+		mempoolCapacity.Set(float64(bc.memPool.capacity))
+		mempoolUnsortedTx.Set(float64(len(bc.memPool.unsortedTxn)))
+		mempoolUnverifiedTx.Set(float64(len(bc.memPool.unverifiedTxn)))
```

Comment:
Maybe wrap these into some kind of structure? They're global and it's not clear at first glance.

Reply:
right now they are in separate go file. Am I right you want to wrap them there and have:
```
BCMetrics struct {
blockHeight
mempoolCapcity
...
}
```
Usage:
```
BCMetrics.updateBlockHeight(<value>)
BCMetrics.updateMempoolCapacity(<value>)
```
?

Reply:
Something like that (not exported from the package, obviously), makes things more clear to me. At the moment these vars look like local while in fact they're not.

Reply:
Something like that (not exported from the package, obviously), makes things more clear to me. At the moment these vars look like local while in fact they're not.